packages/components/src/components/pin-code/pin-code-utils.ts

```diff
@@ -90,13 +90,14 @@ export const syncHiddenInput = (
   required: boolean
 ): void => {
   setAttributes(hiddenInput, {
-    name,
+    ...(name && { name }),
```

Comment:
Did this produce an attribute of undefined or something else? 
If yes, then unit tests are probably incomplete or wrong?